development/print_db_data.py

```diff
@@ -289,7 +290,7 @@ def print_settler_needs():
 		level = line_data.get("level", [-1])
 		for l in level:
 			per_level[l].extend( [ res for (res, num) in line_data[u'consumes'] ] )
-	data = dict( (k, sorted(db.get_res_name(i) for i in v)) for k,v in per_level.iteritems())
+	data = dict( (k, sorted(db.get_res_name(i) for i in v)) for k, v in per_level.iteritems())
```

Comment:
That's an existing problem and should be handled separately, because there are multiple occurrences of that in the codebase.